src/components/application_manager/test/commands/mobile/slider_test.cc

```diff
@@ -279,7 +279,12 @@ TEST_F(SliderRequestTest, OnEvent_UI_OnResetTimeout_UNSUCCESS) {
   (*msg_)[am::strings::msg_params][am::strings::timeout] = kDefaultTimeout;
   (*msg_)[am::strings::params][am::strings::correlation_id] = kCorrelationId;
 
-  CommandPtr command(CreateCommand<SliderRequest>(msg_));
+  // Need to pass 0 for the timeout argument, because the SliderRequest
+  // "Init" method will add the message's "timeout" parameter to the
+  // default timeout that is used in CreateCommand, which causes the
+  // test below to fail. Instead of caling "updateRequestTimeout"
+  // with 1000, the timeout parameter in the event ends up as 1100
+  CommandPtr command(CreateCommand<SliderRequest>(0, msg_));
```

Comment:
This PR includes a fix for invalid memory access which is available at https://github.com/smartdevicelink/sdl_core/pull/2136. With this change, one of the unit tests fails since its implementation hasn't been correct. (Please refer to https://github.com/smartdevicelink/sdl_core/pull/2136 for the details.) We didn't like the unit test to be failing, so we also included a fix for it.

Again, these changes are unrelated to multiple transports so I am OK to remove them. Probably I will file a dedicated issue for unit test failure. By the way, I'm hoping such memory issues are merged to `develop` branch soon, so that I can merge them into this branch without affecting the PR itself.

Reply:
Oh, I didn't notice that the issue has been already resolved with commit 669b0066752959015f75b031bc31a7ede7d8f737. So I just removed this change with commit 634c1ad6a67ee4323362ab376b4f73c4b2a84280.